src/buttons/__tests__/Button.js

```diff
@@ -43,6 +43,19 @@ describe('Button Component', () => {
     expect(onPress).toHaveBeenCalled();
   });
 
+  it('should not be call onPress events when loading is true', () => {
+    const onPress = jest.fn();
+    console.log = jest.fn();
+    const wrapper = shallow(<Button theme={theme} />);
+
+    wrapper.setProps({ onPress, loading: true });
```

Comment:
No need to put this here just set it on the button directly. Also, remove the console.log mock